internal/nodes/validator.go

```diff
@@ -361,14 +365,31 @@ func (v *Validator) findTargetPort(serviceName, ns string, servicePort int32, ke
 	}
 	for _, port := range svcObj.Spec.Ports {
 		// Iterate the ports and find the match for targetPort
-		if servicePort == port.Port {
-			utils.AviLog.Infof("key: %s, msg: Found targetPort %v for Port: %v", key, port.TargetPort.IntVal, servicePort)
+		if serviceBackendPort.Number == port.Port ||
+			serviceBackendPort.Name == port.Name {
+			utils.AviLog.Infof("key: %s, msg: Found targetPort %v for Port: %v", key, port.TargetPort.IntVal, serviceBackendPort)
```

Comment:
@swathinsankaran : Are we intended print whole serviceBackendPort struct or just serviceport  number

Reply:
The user can give either port number or port name in the ingress config. In both cases, we will see at least one value in the log.